lib/SILOptimizer/Mandatory/TFLowerGraph.cpp

```diff
@@ -1630,16 +1649,68 @@ bool TFGraphLowering::copyGraphFunctions(const std::vector<char> &graphDefProto,
     }
   };
 
+  // TODO: enable `foundFunc` related logic when the TF_GetFunctionName() API is
+  // added to TF. bool foundFunc = false;
   for (auto *func : funcs) {
+#if 0
```

Comment:
Please clean up the #if 0 code and the commented out stuff just below

Reply:
This is meant to be enabled once I submit a TF patch to implement a `TF_GetFunctionName()` API. I removed the code here for now.